packages/create-app/index.js

```diff
@@ -4,34 +4,123 @@
 const fs = require('fs')
 const path = require('path')
 const argv = require('minimist')(process.argv.slice(2))
+// eslint-disable-next-line node/no-restricted-require
```

Comment:
**chore:** This line seems not be needed
It doesn't show an error on my machine if I remove it

Reply:
It still warns on my editor 😔. Looks like the rule is configured to read Vite's deps
https://github.com/vitejs/vite/blob/34bd6282c012298a8aa2c82e8300a50a2e3e7b1f/.eslintrc.js#L31-L31

Reply:
Yeah today I found out that some lint errors only show if you haven't build the project and e.g. the `dist` folder is not present